.travis.yml

```diff
@@ -1,9 +1,22 @@
 language: python
 python:
-  - 2.7
```

Comment:
In that case, looks good to me. When the tests pass, I can merge (looks like there's a bad interpreter right now).